rules/ssh-events.rules

```diff
@@ -0,0 +1,10 @@
+# SSH app layer event rules
+#
+# SID's fall in the 2228000+ range. See https://redmine.openinfosecfoundation.org/projects/suricata/wiki/AppLayer
```

Comment:
can you update this wiki page to match this?

Reply:
Done